client/components/tinymce/plugins/wpcom-view/views/simple-payments/index.jsx

```diff
@@ -51,7 +52,9 @@ class SimplePaymentsView extends Component {
 						<Gridicon icon="cross" />
 					</div>
 					<p className="wpview-type-simple-payments__unsupported-message">
-						{ translate( "Your plan doesn't include Simple Payments." ) }
+						{ isEnabled( 'earn/pay-with-paypal' )
+							? translate( "Your plan doesn't include Pay with PayPal" )
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.